presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculator.java

```diff
@@ -53,13 +61,6 @@ public void setUp()
                 .setHighValue(10.0)
                 .setNullsFraction(0.25)
                 .build();
-        SymbolStatsEstimate yStats = SymbolStatsEstimate.builder()
```

Comment:
What is the point of this refactor?

Reply:
so that one can write ... symbolAssertio.isEqualTo(yStats)